### PR TITLE
Update white-label-apps.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/getting-started/white-label-apps.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You should try any scripts on your local machine to test things like authenticat
 
 If you are building many different versions, then instead of creating a long codemagic.yaml file with tens of workflows, you can have a single workflow and use the Codemagic API to pass environment variables to change build parameters.
 
-Users can start new builds using [Codemagic REST API](../rest-api/codemagic-rest-api.md). All data is sent and received as JSON. The REST API supports an optional parameter for environment variables. 
+Users can start new builds using [Codemagic REST API](https://docs.codemagic.io/rest-api/applications/). All data is sent and received as JSON. The REST API supports an optional parameter for environment variables.
```
Comment (helinanever) [inline content/getting-started/white-label-apps.md:61]: Can you use a relative link instead? This should look something like this: /rest-api/applications/

**Review by helinanever — CHANGES_REQUESTED** [review]

File: content/getting-started/white-label-apps.md (hunk 1 above)
Comment (icarusdust) [reply content/getting-started/white-label-apps.md:61]: Relative path gives an empty page.

Comment (helinanever) [reply content/getting-started/white-label-apps.md:61]: Did you test it locally? Works for me.

Comment (nevercode-kevin) [reply content/getting-started/white-label-apps.md:61]: The problem is that the relative link has `.md` on the end. It should be `../rest-api/codemagic-rest-api/`

Comment (icarusdust) [reply content/getting-started/white-label-apps.md]: Changed it!

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**